### ISSUE TITLE
No dependency declared on wcwidth

### ISSUE THREAD

**Issue by lstep** [dbcli/pgcli#462]
pgcli installs correctly, but when trying to run it (first time), complains that wcwidth package hasn't been installed. Using version 0.20.1 installed on Ubuntu Trusty with `pip install pgcli`.

```shell
root@foo:/tmp# pgcli -U user db
Traceback (most recent call last):
  File "/usr/local/bin/pgcli", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 2749, in <module>
    working_set = WorkingSet._build_master()
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 444, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 725, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python2.7/dist-packages/pkg_resources.py", line 628, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: wcwidth
```

**Comment by amjith** [comment]
That's surprising. One of the dependencies of pgcli is prompt-toolkit which has wcwidth as a dependency. But I'll make changes to the setup.py to add wcwidth dependency explicitly. 

Thanks for reporting.

**Comment by amjith** [comment]
This is now available in v1.0. Please upgrade. 

```
pip install -U pgcli
```
